DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/sectionWorkSitesController.js

```diff
@@ -298,6 +297,12 @@ module.exports = function(ngModule) {
       }
     });
 
+    // Employee Data Example Views
+    this.showExampleView = function(view) {
+      vm.exampleView = view;
+      $document[0].getElementById('exampleFirstFocus').focus();
+    }    
```

Comment:
This function should get a new test.  It should assert that `vm.exampleView` is set correctly and that the targeted element ends up with focus.